modules/fluid_properties/src/fluidproperties/TabulatedFluidProperties.C

```diff
@@ -76,6 +76,10 @@ TabulatedFluidProperties::validParams()
       false,
       "Option to use a base-10 logarithmically-spaced grid for specific volume instead of a "
       "linearly-spaced grid.");
+  param.addParam<bool>("p_h_variables",
+                       false,
+                       "If using pressure enthalpy variables, interpolates from PT to h"
+                       "may be wrong especially in multiphase.");
```

Comment:
say what it does?

Reply:
addressed in [a6a0e70](https://github.com/idaholab/moose/pull/27506/commits/a6a0e70fd320f10a2453e722ddc0c69452dc09be)